Marlin/Marlin_main.cpp

```diff
@@ -13564,12 +13563,9 @@ void idle(
   #endif
 
   #if ENABLED(I2C_POSITION_ENCODERS)
-    if (planner.blocks_queued() &&
-        ( (blockBufferIndexRef != planner.block_buffer_head) ||
-          ((lastUpdateMillis + I2CPE_MIN_UPD_TIME_MS) < millis())) ) {
-      blockBufferIndexRef = planner.block_buffer_head;
+    if (planner.blocks_queued() && PENDING(millis(), nextUpdateMillis)) {
```

Comment:
@jaysonkelly I want to make sure this is what you intended. `PENDING` will evaluate true continuously under this setup. I believe this should be `ELAPSED`.